backend/battles/views.py

```diff
@@ -39,6 +39,11 @@ class BattleDetail(DetailView):
 class BattleList(ListView):
     model = Battle
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["logged_user"] = self.request.user
```

Comment:
No need for this. The logged user is automatically included in the templates: https://docs.djangoproject.com/en/3.1/ref/templates/api/#django-contrib-auth-context-processors-auth

So is the request object: https://docs.djangoproject.com/en/3.1/ref/templates/api/#django-template-context-processors-request (so you could do `request.user` in any case)